src/simplify_algebra.cpp

```diff
@@ -185,6 +187,41 @@ struct find_mul_add
     }
 };
 
+// find gelu (a multi-step formula), using the tanh implementation. Replace it with the gelu_erf
+// formula
+struct find_gelu
+{
+    // compile option fast_math controls whether to use math shortcuts like this
+    bool fast_math = true;
```

Comment:
This variable is not used. It also should be `false` by default. You need to check this in `apply` and return early if its false.